__tests__/nc_news.test.js

```diff
@@ -227,4 +227,79 @@ describe('GET /api/articles', () => {
 				});
 		});
 	});
+});
+
+describe('PATCH /api/articles/:article_id', () => {
+	describe('Endpoint Behaviour', () => {
+		test('GET:200 expects correct status code', () => {
+			return request(app)
+				.patch('/api/articles/1')
+				.send({ inc_votes: 100 })
+				.expect(200);
+		});
+		test('GET:200 expects response of updated article', () => {
+			return request(app)
+				.patch('/api/articles/1')
+				.send({ inc_votes: 100 })
+				.then(({ body }) => {
+					const article = body.article;
+					const articleFormat = {
+						title: expect.any(String),
+						topic: expect.any(String),
+						author: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+						article_img_url: expect.any(String),
+					};
+					expect(article).toMatchObject(articleFormat);
+				});
+		});
+		test('GET:200 expects vote count to be updated', () => {
```

Comment:
I like this test but maybe change the description to reflect that you're testing a negative number